### ISSUE TITLE
POM for version 7.9 on Maven Central references "com.dorkbox:ObjectPool:v2.5" which can't be resolved (due the the v before 2.5)

### ISSUE THREAD

**Issue by cheneysan** [dorkbox/TweenEngine#25]

**Comment by cheneysan** [comment]
As a workaround, in Gradle at least, including the ObjectPool dependency first allows it to work:

```
compile "com.dorkbox:ObjectPool:2.5"
compile "com.dorkbox:TweenEngine:7.9"
```

**Comment by dorkbox** [comment]
This is fixed in `7.11`. Thank you for reporting this issue!
